flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -195,9 +196,24 @@ public FlowCheckResponse getFlowState(String flowId) {
         flowCheckResponse.setFlowId(flowId);
         flowCheckResponse.setHasActiveFlow(!completed("Flow", flowId, List.of(), allByFlowIdOrderByCreatedDesc));
         flowCheckResponse.setLatestFlowFinalizedAndFailed(isFlowInFailedState(allByFlowIdOrderByCreatedDesc, failHandledEvents));
+        setStateEventFlowTypeOnFlowCheckResponse(flowCheckResponse, allByFlowIdOrderByCreatedDesc,"flow", flowId);
         return flowCheckResponse;
     }
 
+    public void setStateEventFlowTypeOnFlowCheckResponse(FlowCheckResponse flowCheckResponse, List<FlowLogWithoutPayload> relatedFlowLogs, String type, String id) {
```

Comment:
Please rename the `type` parameter to `marker`, as type can be confusing and in the completed method we already call it that way.

Reply:
@lturcsanyi Renamed it to be `marker`. Thank you!